src/transaction/log_recovery.c

```diff
@@ -4164,15 +4306,17 @@ log_recovery_finish_postpone (THREAD_ENTRY * thread_p, LOG_TDES * tdes)
 	}
 
       if (tdes->coord == NULL)
-	{			/* If this is a local transaction */
-	  (void) log_complete (thread_p, tdes, LOG_COMMIT, LOG_DONT_NEED_NEWTRID, LOG_NEED_TO_WRITE_EOT_LOG);
+	{
+	  LOG_LSA finish_lsa;
+	  log_append_finish_postpone (thread_p, tdes, &finish_lsa);
```

Comment:
finish_postpone should be appended only for the case of pp. 
I want something (safe guard, refactor, rename ...) to make it sure.

Reply:
added non postpone assert